smoke-tests/spec/error_page_spec.rb

```diff
@@ -12,11 +12,142 @@
       }.to raise_error(OpenURI::HTTPError, "404 Not Found")
     end
 
-    xit "serves a 404 response" do
-      URI.open(url)
-    rescue OpenURI::HTTPError => e
-      body = e.io.string
-      expect(body).to eq("404 page not found\n")
+    it "serves a 404 response" do
+      begin
+        URI.open(url)
+      rescue OpenURI::HTTPError => e
+        body = e.io.string
+        expect(body).to include("Error, service unavailable - GOV.UK")
+      end
     end
   end
 end
+
+
+describe "Namespace with no custom default-backend and ingress annotations, serve cluster default-backend error page" do
+  let(:namespace) { "def-backend-#{readable_timestamp}" }
```

Comment:
Objects created by the tests should be named "integrationtest-*"

This makes it a lot easier to find and delete them, if anything gets left behind (e.g. because a test run crashes)